apps/countdown_letters/views.py

```diff
@@ -40,23 +40,20 @@ def results_screen(request):
     letters_chosen: str = request.GET["letters_chosen"]
     players_word: str = request.GET["players_word"]
 
-    valid_word = validations.is_in_oxford_api(players_word)
```

Comment:
Yes, I appreciate this, however I'm not realistically interested in validating whether the word is a correct word or not. I have more exciting projects to spend my time on now.

Initial thoughts to achieve this would be to either use an alternative API which is cheaper or potentially use scraping to find definitions of words, etc.